composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/common/navigation/NavigationRoot.kt

```diff
@@ -50,33 +53,31 @@ fun NavigationRoot() {
             }
 
             entry<Route.Home> {
-                val viewModel = remember {
-                    HomeViewModel(
-                        router = { destination ->
-                            when (destination) {
-                                HomeDestination.NavigateToTimer ->
-                                    backStack.add(element = Route.Timer)
-                            }
+                val viewModelFactory = HomeViewModelFactory(
+                    router = { destination ->
+                        when (destination) {
+                            HomeDestination.NavigateToTimer ->
+                                backStack.add(element = Route.Timer)
```

Comment:
If the ViewModel didn't have a parameter, we wouldn't need a `XViewModelFactory` and could just use `viewModel()`. We will probably always need parameters in our ViewModels, though.